solr-plugin/src/test/java/org/entando/entando/plugins/jpsolr/web/content/AdvContentSearchControllerTest.java

```diff
@@ -615,6 +612,10 @@ void testLoadOrderedPublicEvents_4() throws Exception {
                 this.contentManager.removeOnLineContent(masterContent);
                 this.contentManager.deleteContent(masterContent);
             }
+            synchronized (this) {
```

Comment:
Also in unit tests a "sleep" is a symptom of a synch problem, are you sure it works?

Reply:
@entgigi According with @nshaw I've created a ticket restore the junit test for that cases, the tesst works locally but not in PR